components/autogen/src/format-pages.txt

```diff
@@ -1982,19 +1981,19 @@ metadataRating = Very good
 opennessRating = Fair
 presenceRating = Outstanding
 utilityRating = Poor
-reader = NativeQTReader
+reader = NativeQTReader, LegacyQTReader
 writer = QTWriter
 notes = Bio-Formats has two modes of operation for QuickTime: \n
 \n
-* QTJava mode requires `QuickTime <http://www.apple.com/quicktime/download/>`_ to be \n
-  installed (32-bit JVM only, not supported with 64-bit). \n
+* The Legacy QTJava mode requires QuickTime for Java to be installed \n
+  (no longer available from Apple) and is 32-bit JVM only, not supported with 64-bit. \n
```

Comment:
The end of this sentence is a bit clunky (as it was originally):

"...and is 32-bit JVM only, not supported with 64-bit."

I assume this means that QuickTime for Java needs a 32-bit JVM and won't run with a 64-bit JVM? If so I wonder if it could be phrased a little better?